packages/block-editor/src/components/url-popover/index.js

```diff
@@ -71,6 +71,8 @@ const URLPopover = forwardRef(
 		return (
 			<Popover
 				ref={ ref }
+				role="dialog"
```

Comment:
Is this something we should consider adding in the Popover component? According to [mdn](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/dialog_role) it seems probably okay, but I'm not sure if that was a conscious decision to leave the consumers handle it.

Additionally we have some Popover usages that fit the mdn description, but not 100% sure. Example is the in-between inserter.. 

--cc @mirka @afercia

Reply:
That's what happens when we let components scope creep. Not blaming anyone, just a statement of reflection we've all got to live with now. I think given the situation we're in, might have to manually add it. :(

Reply:
Popover is used for a lot of things, so I'm not sure it should be added everywhere. For example, I don't think `role=dialog` is supposed to be added to dropdown menus, which already have the `menu` role.

Perhaps if there were more distinct components, like `DialogPopover`, beyond that I'm not sure there's an easy way to enforce it.